service/frontend/fx.go

```diff
@@ -642,6 +642,9 @@ func NexusHTTPHandlerProvider(
 	metricsHandler metrics.Handler,
 	namespaceRegistry namespace.Registry,
 	authInterceptor *authorization.Interceptor,
+	namespaceRateLimiterInterceptor *interceptor.NamespaceRateLimitInterceptor,
```

Comment:
Are these rate limit interceptors used? Or just planning to add them in a future PR?

Reply:
Next PR.